sympy/series/formal.py

```diff
@@ -1210,6 +1219,77 @@ def __rmul__(self, other):
         return self.__mul__(other)
 
 
+class FiniteFormalPowerSeries(FormalPowerSeries):
+    """Represents Finite Formal Power Series of a function.
+
+    No computation is performed. This class should only to be used to represent
+    a series. No checks are performed.
+
+    The difference between `FiniteFormalPowerSeries` and `FormalPowerSeries` is that
+    the coefficient sequence of a `FiniteFormalPowerSeries` is a list of coefficients,
+    instead of a sequence as in a `FormalPowerSeries` object.
+
+    `product`, `compose`, and `inverse` functions of `FormalPowerSeries` class, return
+    an object of this class.
+
+    See Also
+    ========
+
+    sympy.series.formal.FormalPowerSeries
+    sympy.series.formal.fps
+    """
+    def __new__(cls, *args):
+        args = map(sympify, args)
+        return Expr.__new__(cls, *args)
+
+    def __init__(self, *args):
+        pass
+
+    def _eval_term(self, pt):
```

Comment:
What is this function is supposed to do? I thought computing ak coefficient point wise not possible?